src/components/Contributors.js

```diff
@@ -1,59 +1,43 @@
 //import { Link } from "react-router-dom";
 import Axios from "axios";
-import React, { useState, useEffect, usePrevious} from "react";
+import React, {useEffect, usePrevious, useState} from "react";
 import ContributorsList from "./ContributorsList";
 
 
-
 export default function Contributors(props) {
-
     const [contributorsData, setContributorsData] = useState();
 
-
-
     const getContributorsData = (e) => {
-       Axios.get(`http://localhost:5000/contributors/`)
-            .then((response) => {
+        Axios.get(`/api/contributors/`)
+            .then(response => {
                 setContributorsData(response.data)
-               console.log('GETresponse',response)
-
             })
-
-            .catch((error) => console.log(error))
+            .catch(console.error)
     }
-    useEffect(()=>{
-        getContributorsData();
 
-    },[])
+    useEffect(() => {
+        getContributorsData();
+    }, [])
 
-    
     return (
-    
-            <section className="credits">
-                <h2 className="credits__heading">Contributors</h2>
-                <div className="credits__contributors-wrapper">
-                    {contributorsData && contributorsData.map((contributorsData) => {
-                        return (
-                            <ContributorsList
-                                key={contributorsData.id}
-                                id={contributorsData.id}
-                                name={contributorsData.name}
-                                image={contributorsData.image}
-                                linkedIn={contributorsData.linkedIn}
-                                gitHub={contributorsData.gitHub}
-                                twitter={contributorsData.twitter}
-                            />
-                            
-                        )
-                    })}
-  
-                
-                </div>
-              {usePrevious}
-            </section>
-
-
-       
+        <section className="credits">
+            <h2>Contributors</h2>
+            <div className="d-flex flex-wrap">
+                {contributorsData && contributorsData.map(contributorsData => {
+                    return (
+                        <ContributorsList
+                            key={contributorsData.name}
+                            name={contributorsData.name}
+                            image={contributorsData.image}
+                            linkedIn={contributorsData.linkedIn}
+                            gitHub={contributorsData.gitHub}
+                            twitter={contributorsData.twitter}
+                        />
+
+                    )
+                })}
+            </div>
+            {usePrevious}
```

Comment:
I left this in but it feels redundant when there's no real state change?

Reply:
This was the only way I could find to stop the infinite loop.